pkg/monitoring/alloy/templates/alloy-config.alloy.template

```diff
@@ -3,13 +3,17 @@ logging {
   format = "logfmt"
 }
 
-prometheus.operator.servicemonitors "default" {
-  forward_to = [prometheus.remote_write.default.receiver]
+prometheus.operator.servicemonitors "giantswarm_legacy" {
```

Comment:
So the `application.giantswarm.io/team` label in servicemonitors is definitely legacy?
We drop the idea of having a `team` label added to the metrics?

Reply:
I do not see the 2 has being the same though.

I think it being used as a service discovery mechanism (on the service monitor) is gone. But I still see value in it being a pod label or even a resource labeel because then it's easy to know who owns this clusterrole and so on

Reply:
Do we actually convert the smon label to a metrics label? Did we ever do it?
I'm confused with where metrics' team label can come from, and don't see the relationship with the service/pod monitors metadata labels. :thinking: 
So my remark probably didn't make any sense.

Reply:
No it does make sense. We do not have the team in our own metrics today but we have it in some of the (a.k.a. the app exporter metrics)

App exporter extracts the value from the annotation on the Chart.yaml